src/components/v5/common/Fields/Switch/Switch.module.css

```diff
@@ -0,0 +1,13 @@
+.toggle::after {
+  @apply absolute top-0.5 left-0.5 bg-base-white rounded-full h-4 w-4 transition shadow-sm;
```

Comment:
Please, use tailwind rules. You have an example i src/components/v5/common/Checkbox/Checkbox.module.css how you can do it ('peer' rule) 

https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state